lms/static/js/spec/main.js

```diff
@@ -316,7 +319,11 @@
             },
             'js/ccx/schedule': {
                 exports: 'js/ccx/schedule',
-                deps: ['jquery', 'underscore', 'backbone', 'gettext']
+                deps: ['jquery', 'underscore', 'backbone', 'gettext', 'moment'],
+                init: function() {
+                    // Set global variables that the schedule code is expecting to be defined
+                    window.moment = require('moment');
```

Comment:
Ugh, is there any other way you can do this? Can you call `require()` within your `schedule.js` file? This is really ugly and crufty.